### PR TITLE
Example of not showing the accessibility options menu

### PR DESCRIPTION
**Pull request by orenco-resonai**
Related issue: https://github.com/resonai/resonai2/issues/16103

Related PR: https://github.com/resonai/resonai2/pull/16179

### CODE REVIEW

File: sample-apps/communication/main.js [hunk 1]
```diff
@@ -70,12 +70,15 @@ async function init () {
   document.getElementById('navigate-message-nopreview-button').addEventListener('click', () => {
     onNavigateUsingMessageClick({ skipPreview: true })
   })
+  document.getElementById('navigate-message-noaccessibility-button').addEventListener('click', () => {
+    onNavigateUsingMessageClick({ noaccessibility: true })
```
Comment (bergden-resonai) [inline sample-apps/communication/main.js:74]: noAccessibility

File: sample-apps/communication/main.js [hunk 2]
```diff
@@ -84,6 +87,10 @@ function onNavigateUsingMessageClick ({ skipPreview = false } = {}) {
     packageName: communicationSamplePackageName,
     actions: { navigationSuccess: true, navigationCancelled: true }
   }
+  if (noaccessibility) {
+    // To not show the accessibility options menu, send a preset option (e.g. 'all' for 'Fastest')
+    data.presetCapability = 'all'
```
Comment (bergden-resonai) [inline sample-apps/communication/main.js:92]: Why is `all` related to `fastest`?

Comment (yoav-resonai) [reply sample-apps/communication/main.js:92]: if we allow all possibilities - we get the fastest way

Comment (orenco-resonai) [reply sample-apps/communication/main.js:92]: When all options are available to you (elevator, stairs, etc.) - the system is free to choose the fastest route

**Review by bergden-resonai — APPROVED** [review]

**Pull request merged**